js/widgets/fixedToolbar.js

```diff
@@ -54,17 +53,22 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 				this._makeFixed();
 			}
 			if ( this.options.position === "fixed" && !this.options.supportBlacklist() ) {
-				var $page = ( !!this.page )? this.page: ( $(".ui-page-active").length > 0 )? $(".ui-page-active"): $(".ui-page").eq(0);
+				var currentPage = ( !!this.page ) ? 
+					this.page : 
+					( $(".ui-page-active").length > 0 ) ? 
+					$(".ui-page-active") : 
+					$(".ui-page").eq(0);
 
 				if ( o.fullscreen !== undefined) {
 					if ( o.fullscreen ) {
-						this.element.addClass( "ui-"+ this.role +"-fullscreen" );
-						$page.addClass( "ui-page-" + this.role + "-fullscreen" );
+						this._addClass( "ui-toolbar-"+ this.role +"-fullscreen" );
+						this._addClass( currentPage,  "ui-page-" + this.role + "-fullscreen" );
 					}
 					// If not fullscreen, add class to page to set top or bottom padding
 					else {
-						this.element.removeClass( "ui-"+ this.role +"-fullscreen" );
-						$page.removeClass( "ui-page-" + this.role + "-fullscreen" ).addClass( "ui-page-" + this.role+ "-fixed" );
+						this._removeClass( "ui-toolbar-"+ this.role +"-fullscreen" );
+						this._removeClass( currentPage, "ui-page-" + this.role + "-fullscreen" );
```

Comment:
This is interesting because ui-page-footer-fullscreen is essentially a toolbar class but its `ui-page-*` so it should in the extra param. Im thinking we may want to rename this.